Traceback (most recent call last):
  File "../main.py", line 100, in <module>
    run(args)
  File "../main.py", line 89, in run
    exec(open("../testermain.py").read()) 
  File "<string>", line 38, in <module>
  File "/rigel/theory/users/ab4877/Ultrametric-benchmark/diagnosis.py", line 73, in hierarchical_error
    outputs = net(samples)
  File "/rigel/home/ab4877/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/rigel/theory/users/ab4877/Ultrametric-benchmark/neuralnet.py", line 99, in forward
    out = F.relu(self.bn1(self.conv1(x)))
  File "/rigel/home/ab4877/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 493, in __call__
    result = self.forward(*input, **kwargs)
  File "/rigel/home/ab4877/.local/lib/python3.5/site-packages/torch/nn/modules/conv.py", line 338, in forward
    self.padding, self.dilation, self.groups)
RuntimeError: Given groups=1, weight of size 64 3 3 3, expected input[1, 1, 28, 28] to have 3 channels, but got 1 channels instead
